cli/command/stack/kubernetes/cli.go

```diff
@@ -52,25 +51,27 @@ func WrapCli(dockerCli command.Cli, cmd *cobra.Command) (*KubeCli, error) {
 	if err != nil {
 		return nil, fmt.Errorf("Failed to load kubernetes configuration file '%s'", kubeConfig)
 	}
-	cli.kubeConfig = config
+	cli.KubeConfig = config
```

Comment:
Related to other changes, this "get KubeConfig from flag or env" can be extracted into a function that just takes a single `path string` argument.